src/sentry/api/serializers/rest_framework/organizationmemberinvite.py

```diff
@@ -172,3 +165,10 @@ def validate_approve(self, approve):
             raise serializers.ValidationError(str(err))
 
         return approve
+
+
+class OrganizationMemberReinviteRequestValidator(serializers.Serializer):
+    trigger_regenerate_token = serializers.BooleanField(
+        required=False,
+        help_text="Whether or not to regenerate the token for this invitation",
+    )
```

Comment:
nit: should this be defaulted to `False` so you can do `result["trigger_regenerate_token"]` in the reinvite endpoint?

```suggestion
    trigger_regenerate_token = serializers.BooleanField(
        required=False,
        default=False,
        help_text="Whether or not to regenerate the token for this invitation",
    )
```